src/protocol/modulus_conversion/convert_shares.rs

```diff
@@ -136,6 +136,7 @@ where
 pub async fn convert_all_bits<F, C, S>(
     ctx: &C,
     locally_converted_bits: &[Vec<BitConversionTriple<S>>],
+    // TODO: u8 should be enough to represent bits
```

Comment:
I think it was @martinthomson who said that rust just really wants you to use u32 for bitwise stuff.

Reply:
[`type_of(u64::BITS) == u32`](https://doc.rust-lang.org/std/primitive.u64.html#associatedconstant.BITS)

Seems overkill to me, but I'd prefer to use compatible types than to have to continually think about type conversion.